src/content/docs/dropins/cart/containers/gift-cards.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: GiftCards container
+description: Learn about the GiftCards container.
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+The `GiftCards` container manages the application and removal of gift cards to the shopping cart. It provides a text box for users to enter gift card codes. The container uses the `applyGiftCardToCart` and `removeGiftCardFromCart` functions to apply the coupon to the cart. When a gift card code is applied, the corresponding amount is subtracted from the total order value, and the discount is displayed in the cart and order summary.
+
+<Diagram caption="GiftCards container">
+  ![GiftCards container](@images/dropins/cart/gift-cards.png)
+</Diagram>
+
+## Configurations
+
+The `GiftCards` container provides the following configuration options:
+
+/*
+Note to reviewer: I need help with this section. This container is similar to Coupons, but I don't understand what's going on here.
+*/
```

Comment:
It's might be a bit confusing seems configurations are 90% identical. The only difference - `className`, `accordionSectionTitle` and `accordionSectionIcon`. Rest configurations are identical.